src/collection/views/Collection.tsx

```diff
@@ -41,10 +41,7 @@ import {
 	ToolbarRight,
 } from '@viaa/avo2-components';
 import PermissionGuard from '../../authentication/components/PermissionGuard';
-import {
-	PermissionGuardFail,
-	PermissionGuardPass,
-} from '../../authentication/components/PermissionGuard.slots';
+import { PermissionGuardPass } from '../../authentication/components/PermissionGuard.slots';
```

Comment:
Als je enkel de Pass slot gebruikt mag je de slot ook gewoon weglaten. Dan wordt de content van de PermissionGuard automatisch in de PermissionGuardPass slot gestoken